.gitignore

```diff
@@ -1 +1,2 @@
+composer.lock
```

Comment:
It should not be ignored, we'd like to run test with locked dependencies (see Travis CI configuration in .travis.yml)

Reply:
Got it!